contract-tests/sdkClientEntity.js

```diff
@@ -172,31 +172,47 @@ export async function newSdkClientEntity(options) {
           check: migrationOperation.trackConsistency ? (a, b) => a === b : undefined,
           readNew: async () => {
             try {
-              const res = await got.post(migrationOperation.newEndpoint, {});
+              const options = {};
+              if (migrationOperation.payload) {
+                options.body = migrationOperation.payload;
+              }
+              const res = await got.post(migrationOperation.newEndpoint, options);
```

Comment:
Looks like we can dry this logic with a function:

```ts
        const createMigrationOptions = ({ payload }) => {
          return payload ? { body: payload } : {};
        };
```

```suggestion
              const res = await got.post(
                migrationOperation.newEndpoint,
                createMigrationOptions(migrationOperation),
              );
```

Reply:
I actually realized that I messed this up.